db/migrate/20180410125416_rename_ansible_job_categories.rb

```diff
@@ -7,21 +7,23 @@ def up
     User.as_anonymous_admin do
       updated_templates = ['Power Action - Ansible Default',
                            'Puppet Run Once - Ansible Default']
-      job_templates = JobTemplate.without_auditing.where(
-        :name => updated_templates
-      ).all
-      job_templates.each do |job_template|
-        next if job_template.job_category =~ /^Ansible/
-        job_template.job_category = "Ansible #{job_template.job_category}"
-        job_template.save_without_auditing
-      end
-
-      service_template = JobTemplate.where(
-        :name => 'Service Action - Ansible Default'
-      ).first
-      if service_template.present?
-        service_template.job_category = 'Ansible Services'
-        service_template.save_without_auditing
+      JobTemplate.without_auditing do
+        job_templates = JobTemplate.where(
+          :name => updated_templates
+        ).all
+        job_templates.each do |job_template|
+          next if job_template.job_category =~ /^Ansible/
+          job_template.job_category = "Ansible #{job_template.job_category}"
+          job_template.save_without_auditing
```

Comment:
if you disable auditing using block you don't need to `save_without_auditing` here and below

Reply:
If you don't mind I think I'll merge as it's harmless and some people are already complaining about not being able to migrate

Reply:
sorry I was busy, I opened https://github.com/theforeman/foreman_ansible/pull/168 to address that